src/blocks/kpi-chart/index.tsx

```
@@ -0,0 +1,359 @@
+	switch (periodType) {
+		case "week":
+			// Next period starts the day after the current Sunday
+			nextStart.setDate(nextStart.getDate() + 1);
+			break;
+		case "month":
+			// Next period starts on the 1st of the next month
+			nextStart.setMonth(nextStart.getMonth() + 1, 1);
+			break;
+		case "quarter":
+			// Calculate the next quarter's start month
+			const currentMonth = nextStart.getMonth();
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)